shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profilePreferences/presentation/profile/components/ProfileLettermark.kt

```diff
@@ -0,0 +1,37 @@
+package com.vml.tutorial.plantshop.profilePreferences.presentation.profile.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import com.vml.tutorial.plantshop.profilePreferences.domain.User
+import com.vml.tutorial.plantshop.ui.theme.Typography
+
+@Composable
+fun ProfilePhoto(
+    user: User?,
+    textStyle: TextStyle = Typography.displayMedium,
+    modifier: Modifier = Modifier
+) {
+    val roundedCornerShape = remember { RoundedCornerShape(16.dp) }
+    Box(
```

Comment:
Can you try something? If we use IconButton, does it cut the ripple effect in a rectangular shape?
Can you try to achieve to have a rounded ripple effect or click effect there ?

![image](https://github.com/kgeriiie/plant-shop/assets/10209796/a4e17a09-825d-4af3-811d-f7425f58b887)

Reply:
I tried the IconButton and the ripple effect had the same shape as the round button. I resolved the issue by refactoring the modifier because the same component is used in the profile ProfileScreen and it is not clickable. If we disable the IconButton, it will be greyed out.

Reply:
You made it :) cool :) one little thing :) If I log in to the app, it shows me a null-null value for 1 sec until it retrieves the user data. Can you show an empty string instead of null null if we don't have the right value?